The-SMIRNOFF-force-field-format.md

```diff
@@ -1,103 +1,288 @@
+This method is provided solely for convenience in exploring alternative charging schemes; actual forcefield releases for distribution will use one of the other mechanisms specified above.
+
+**QUESTION:** Do we still want to support being able to pull charges from `OEMol`s for convenience, or is the `<LibraryCharges>` mechanism sufficiently general?
 
 ## Parameter sections
 
 For this section it will help to have on hand an example SMIRNOFF file, such as that the [AlkEthOH example ffxml](https://github.com/openforcefield/openforcefield/blob/master/openforcefield/data/forcefield/Frosst_AlkEthOH.ffxml) or the larger prototype [SMIRNOFF99Frosst ffxml](https://github.com/openforcefield/SMIRNOFF99Frosst/blob/master/SMIRNOFF99Frosst.ffxml).
 
-Before getting in to individual sections, it's worth noting that the XML parser ignores attributes in the XML that it does not understand, so providing a parameter line for an angle that specifies (for example) a second force constant `k2` will lead to no effect.
-
-### NONBONDED PARAMETERS (E.G. LENNARD-JONES)
+### `<vdWForce>`
 
-Nonbonded parameters (currently, Lennard-Jones parameters) are specified via the [`NonbondedForce`](http://docs.openmm.org/7.0.0/userguide/theory.html#nonbondedforce) tag with sub-tags for individual `Atom` entries, such as:
+van der Waals force parameters, which include repulsive forces arising from Pauli exclusion and attractive forces arising from dispersion, are specified via the `StericsForce` tag with sub-tags for individual `Atom` entries, such as:
 ```XML
-<NonbondedForce coulomb14scale="0.833333" lj14scale="0.5" sigma_unit="angstroms" epsilon_unit="kilocalories_per_mole">
+<vdWForce potential="Lennard-Jones" scale12="0.0" scale13="0.0" scale14="0.5" scale15="1" sigma_unit="angstroms" epsilon_unit="kilocalories_per_mole">
    <Atom smirks="[#1:1]" rmin_half="1.4870" epsilon="0.0157"/>
    <Atom smirks="[#1:1]-[#6]" rmin_half="1.4870" epsilon="0.0157"/>
    ...
-</NonbondedForce>
+</vdWForce>
+```
+Attributes in the `<vdWForce>` tag specify the scaling terms applied to the energies of 1-2 (`scale12`, default: 0), 1-3 (`scale13`, default: 0), and 1-4 (`scale14`, default: 0.5) interactions, (`scale15`. default: 1.0).
+
+Currently, only `potential="Lennard-Jones-12-6"` is supported:
+```
+U(r) = 4*epsilon*((sigma/r)^12 - (sigma/r)^6)
+```
+Later revisions will add support for additional potential types (e.g., `Buckingham-exp-6`) and the ability to support arbitrary algebraic functional forms.
+If the `potential` attribute is omitted, it defaults to `Lennard-Jones-12-6`.
+
+**QUESTION:** We need to specify mixing rules as well. How should we do this? Via a single attribute like `mixing_rules="Lorentz-Berthelot"`, via multiple attributes like `sigma_mixing_rule="geometric" epsilon_mixing_rule="arithmetic"`, or via a block like
```

Comment:
At first the block seemed like overkill, but it does allow better flexibility. So maybe that's best.

Reply:
I agree with David